librosa/core/constantq.py

```diff
@@ -55,10 +55,13 @@ def cqt(y, sr=22050, hop_length=512, fmin=None, n_bins=84,
         Number of bins per octave
 
     tuning : None or float in `[-0.5, 0.5)`
-        Tuning offset in fractions of a bin (cents).
+        Tuning offset in fractions of a bin.
 
         If `None`, tuning will be automatically estimated from the signal.
 
+        The minimum frequency of the resulting CQT will be modified to
+        `fmin * 2**(tuning / bins_per_octave)`.
```

Comment:
What is the use case for setting `tuning` as a nonzero value that isn't estimated from `y`?

Reply:
Perhaps you have a different tuning estimator?